### PR TITLE
docs: Adds version numbers to quickstart module calls

### PR DESCRIPTION
**Pull request by vikram-dagger**
...and updates the arguments to the Go builder module

### CODE REVIEW

File: docs/versioned_docs/version-zenith/quickstart/883939-containers.mdx [hunk 1]
```diff
@@ -18,7 +18,7 @@ Just as you can chain and return `Directory` types, you can also chain and retur
 Try calling this function:
 
 ```shell
-dagger call -m github.com/shykes/daggerverse/wolfi base with-package --name cowsay container
+dagger call -m github.com/shykes/daggerverse/wolfi@7884cbc1941f85a7c714fc7f1bcc5c148842c21e base with-package --name cowsay container
```
Comment (shykes) [inline docs/versioned_docs/version-zenith/quickstart/883939-containers.mdx:21]: Let me add a version number real quick

File: docs/versioned_docs/version-zenith/quickstart/883939-containers.mdx [hunk 2]
```diff
@@ -43,7 +43,7 @@ One of the most interesting `Container` functions is `Terminal()`, which can be
 To see this in action, call the previous function again, this time chaining an additional function call to `Shell()` on the returned `Container`:
```
Comment (shykes) [inline docs/versioned_docs/version-zenith/quickstart/883939-containers.mdx:43]:
```suggestion
To see this in action, call the previous function again, this time chaining an additional function call to `terminal()` on the returned `Container`:
```

File: docs/versioned_docs/version-zenith/quickstart/883939-containers.mdx [hunk 3]
```diff
@@ -76,15 +76,15 @@ While most terminal programs such as `htop` or `vim` work with `dagger ... termi
 The `Container` type has a `WithExec()` function, which returns the container after executing a specific command inside it. So, you could achieve the same result as before (although non-interactively) by chaining a function call to `WithExec()` on the `Container` returned previously, as shown in the following command:
```
Comment (shykes) [inline docs/versioned_docs/version-zenith/quickstart/883939-containers.mdx:76]:
```suggestion
The `Container` type has a `withExec()` function, which returns the container after executing a specific command inside it. So, you could achieve the same result as before (although non-interactively) by chaining a function call to `WithExec()` on the `Container` returned previously, as shown in the following command:
```

File: docs/versioned_docs/version-zenith/quickstart/883939-containers.mdx [hunk 4]
```diff
@@ -76,15 +76,15 @@ While most terminal programs such as `htop` or `vim` work with `dagger ... termi
...
 
 ```shell
-dagger call -m github.com/shykes/daggerverse/wolfi base with-package --name=cowsay container with-exec --args "cowsay" --args "dagger" stdout
+dagger call -m github.com/shykes/daggerverse/wolfi@7884cbc1941f85a7c714fc7f1bcc5c148842c21e base with-package --name=cowsay container with-exec --args "cowsay" --args "dagger" stdout
```
Comment (shykes) [inline docs/versioned_docs/version-zenith/quickstart/883939-containers.mdx:79]: dagger call -m github.com/shykes/daggerverse/wolfi@v0.1.2 container --packages=cowsay with-exec --args cowsay,dagger stdout

Comment (shykes) [inline docs/versioned_docs/version-zenith/quickstart/883939-containers.mdx:79]:
```suggestion
dagger call -m github.com/shykes/daggerverse/wolfi@v0.1.2 container --packages=cowsay with-exec --args cowsay,dagger stdout
```

**Review by sipsma — APPROVED** [review]

**Pull request merged**